File: Back.DAL/Models/SuccessfulLoginResponseModel.cs
using System;

namespace Back.DAL.Models
{
    /// <summary>
    /// خروجی متد ورود موفق به سیستم
    /// </summary>
    public class SuccessfulLoginResponseModel
    {
        /// <summary>
        /// شناسه کاربر
        /// </summary>
        public int UserId { get; set; }

        /// <summary>
        /// شناسه جلسه ورود کاربر - یکتا برای کاربر
        /// </summary>
        public Guid SessionId { get; set; }

        /// <summary>
        /// تاریخ و ساعت ورود به سیستم
        /// </summary>
        public DateTime LoginTime { get; set; }

        /// <summary>
        /// توکن امنیتی جلسه
        /// </summary>
        public string Token { get; set; }
    }
}

File: Front.Web/ClientApp/src/app/models/meta/SuccessfulLoginResponseModel.ts
//اطلاعات دریافتی از سرور در صورت لاگین موفق
export class SuccessfulLoginResponseModel {
  constructor(
    public userId: number, //شناسه کاربر در تدبیر
    public sessionId: string, //شناسه جلسه کاربرر -- ممکن است کاربر از چندین کامپیوتر یا مرورگر لاگین شده باشد
    public loginTime: string, //زمان ورود
    public token: string, //توکن امنیتی که برای بازسازی جلسه کاربر مورد نیاز است
    public workspaceId: number, //آخرین شرکت که کاربر به آن وارد شده
    public fpId: number, //آخرین دوره مالی که کاربر به آن وارد شده
    public userRealName: string //نام واقعی کاربر
  ) {}
}
